src/web/src/components/Header/MobileHeader.tsx

```diff
@@ -1,151 +1,90 @@
-import { useState, KeyboardEvent, SyntheticEvent } from 'react';
 import Link from 'next/link';
 import { makeStyles } from '@material-ui/core/styles';
-import { AppBar, Toolbar, IconButton, List, ListItem, Drawer, Divider } from '@material-ui/core';
-import MenuIcon from '@material-ui/icons/Menu';
+import { AppBar, Toolbar, IconButton } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
+import HomeIcon from '@material-ui/icons/Home';
+import ContactSupportIcon from '@material-ui/icons/ContactSupport';
+
+// 1.7 band-aid (Removing theme | line 6)
+// import dynamic from 'next/dynamic';
 
-import Logo from '../Logo';
 import Login from '../Login';
-// 1.7 band-aid (Removing theme | line 11)
-// import ThemeToggleButton from '../ThemeToggleButton';
+
+/**  This will solve the problem of incorrect rendering of theme icon when theme preference is dark
+ * This ensures that the version displayed to user is the client view which ties to the client's preference theme.
+ * This is only an issue on DesktopHeader since on MobileHeader there is a listener triggering rerendering.
+ * */
+// 1.7 band-aid (Removing theme | line 16 - 18)
```

Comment:
Just clean up the commented out code, and it's good! Nice work! 😄

Reply:
@tonyvugithub wrote that comment, maybe he wants in the future? I'll let him decide on this.

Reply:
@Metropass @DukeManh This comment is mine from our 1.7 release.

Reply:
@PedroFonsecaDEV Noted! Approved

Reply:
Do you want to keep it?

Reply:
My comment is only the line `// 1.7 band-aid (Removing theme | line 16 - 18)` 
Let's keep it.